R/parameters.R

```diff
@@ -411,6 +412,26 @@ generate_parameters_new_hospital_model <- function(
   I0_asympt[seed_idx,1,N_trans_classes] <- infection_seeding$values
   S0[seed_idx] <- S0[seed_idx] - infection_seeding$values
   
+  if (hospital_fitted){
+  
+    # Read file with fitted parameters (from Bob Verity's hospital model)
+    fitted_parameters <- readRDS(file = "inst/extdata/fitted_parameters.rds")
```

Comment:
how is this file generated? where is it generated? how is it updated?

Reply:
This file is generated from a spreadsheet, coming from Bob's model and processed by Neil. I have no idea of how it will move forward, but ideally we would like to get this information directly from Bob's ultimately. The fitting is to transform a cum inverse gamma distribution into an Erlang one using KL divergence. The code for this is on the sandbox.
https://github.com/mrc-ide/sircovid-sandbox/pull/2